config/config.go

```diff
@@ -1597,6 +1621,11 @@ func (c *Config) CheckConfig() error {
 		return fmt.Errorf(ErrCheckingConfigValues, err)
 	}
 
+	err = c.checkGCTScriptConfig()
+	if err != nil {
+		log.Errorf(log.Global, "Failed to configure gctscript, feature has been disabled: %s\n", err)
```

Comment:
Maybe we can also change the config value from true to false when this errors, so it doesn't try to starts the script manager? This seems like it would be very rare to ever happen though, if ever.

Reply:
Have not forgotten about this

disabled is most likely not the best wording here since gctscript will never really fail to configure to a point where it is unusable as we set safe defaults in checkGCTScriptConfig() might just set a warning that it has reset to default instead